example/pom.xml

```diff
@@ -67,6 +67,48 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <groupId>org.eclipse.hono</groupId>
       <artifactId>hono-demo-certs</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>broker</artifactId>
```

Comment:
there is no such artifact. It is called `hono-artemis` now ... this currently beaks the build on Travis.